packages/libcunit/build.sh

```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://sourceforge.net/projects/cunit/
+TERMUX_PKG_DESCRIPTION="C Unit Testing Framework"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Thibault Meyer <meyer.thibault@gmail.com>"
+TERMUX_PKG_VERSION=2.1.3
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL=https://github.com/Linaro/libcunit/releases/download/2.1-3/CUnit-2.1-3.tar.bz2
+TERMUX_PKG_SHA256=f5b29137f845bb08b77ec60584fdb728b4e58f1023e6f249a464efa49a40f214
+TERMUX_PKG_BREAKS="libcunit-dev"
+TERMUX_PKG_REPLACES="libcunit-dev"
```

Comment:
Since libcunit-dev has never existed (in our repos), you do not need to specify that the package breaks and replaces it. Other packages have these lines since we used to split header files into separate packages

Reply:
Removed